runtime-config-linux.md

```diff
@@ -144,6 +144,12 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
 
+There are a few facets to this:
+
+* Managing the existence of cgroups within the hierarchy (`cgroupsPath`).
+* Assigning processes to cgroups.
```

Comment:
On Wed, Dec 16, 2015 at 10:23:42AM -0800, Vish Kannan wrote:
> > +* Assigning processes to cgroups.
> 
> nit: Moving processes to respective cgroups.

Done in 2e002e1 → 734205b, where I also rebased onto master.  I prefer
my initial wording, because assigning cgroups doesn't rule out removal
from a previous cgroup (which the kernel handles for us), and I'm not
sure what “respective” is clarifying.  But I don't feel strongly
enough about either to go back and forth on it ;).